admin/jqadm/themes/dashboard.js

```diff
@@ -88,16 +88,16 @@ Aimeos.Dashboard = {
 
 			var svg = d3.select(selector)
 				.append("svg")
-					.attr("width", width + margin.left + margin.right)
-					.attr("height", (width - lgspace > height ? height + margin.top + margin.bottom : radius * 2 + margin.top + data.data.length * txheight + 25 ));
+				.attr("width", width + margin.left + margin.right)
```

Comment:
Ups, you removed the indentions of the other code by accident now ...

Reply:
Yes, stupid :) Changed this.